scripts/build-search-index.js

```diff
@@ -0,0 +1,36 @@
+const fs = require("fs");
+
+const metricIndex = {};
+
+const apps = fs
+  .readdirSync("public/data", { withFileTypes: true })
+  .filter((d) => d.isDirectory())
+  .map((d) => d.name);
+
+apps.forEach((app) => {
+  const metricsDir = `public/data/${app}/metrics`;
+  const metricsFiles = fs.readdirSync(metricsDir);
+  metricsFiles.forEach((mf) => {
+    const metric = JSON.parse(
+      fs.readFileSync(`${metricsDir}/${mf}`).toString()
+    );
+    const globalId = `${app}-${metric.name}`;
+    const metricSummary = {
+      id: metric.name,
+      type: metric.type,
+      tags: (metric.tags || []).map((tag) => tag.name),
+      app,
+      description: metric.description,
+    };
+    metricIndex[globalId] = metricSummary;
+  });
+});
```

Comment:
This step could probably be handled entirely within the existing `build-glean-metadata.py` script, but wrote things up this way for ease of prototyping.